app/renderer/components/navigation/navigationBar.js

```diff
@@ -115,6 +125,77 @@ class NavigationBar extends React.Component {
       this.props.bookmarked
   }
 
+  // BEM Level: navigationBar__navigationButtonContainer
+  get stopButton () {
+    return <button className={cx({
```

Comment:
no need to do now but would be cool if we have here our browserButton with iconOnly prop